pydm/widgets/label.py

```diff
@@ -1,15 +1,16 @@
 from .base import PyDMWidget, TextFormatter, str_types
 from qtpy.QtWidgets import QLabel, QApplication
-from qtpy.QtCore import Qt, Property, Q_ENUMS
+from qtpy.QtCore import Qt, Property
 from .display_format import DisplayFormat, parse_value_for_display
 from pydm.utilities import is_pydm_app, is_qt_designer
 from pydm import config
 from pydm.widgets.base import only_if_channel_set
+from PyQt5.QtCore import Q_ENUM
 
 _labelRuleProperties = {"Text": ["value_changed", str]}
 
 
-class PyDMLabel(QLabel, TextFormatter, PyDMWidget, DisplayFormat, new_properties=_labelRuleProperties):
+class PyDMLabel(QLabel, TextFormatter, PyDMWidget, new_properties=_labelRuleProperties):
```

Comment:
Removing DisplayFormat here (as well as in line_edit.py) will cause issues with saved ui files not being able to find the property in PyQt5, error I am seeing with the example label below.

This is where the idea to set up the enum this way came from
https://stackoverflow.com/questions/46917671/how-to-use-q-enums-in-pyqt



```
$ pydm examples/label/label.ui
<...>
AttributeError: type object 'PyDMLabel' has no attribute 'String'
```